demo/src/app/components/typeahead/overview/sections/basic-usage/template.html

```diff
@@ -0,0 +1,47 @@
+<p>
+  To set up a typeahead instance, you need to pass an initialization function to it. This function must return an <code>Observable</code> spawning lists of options: this is how you control the list of displayed options.
+</p>
+
+<p>
+  Pass it that way on the template side:
+</p>
+
+<ngbd-code [snippet]="snippets.observableSetup.template"></ngbd-code>
+
+<p>
+  And define it that way on the component side:
+</p>
+
+<ngbd-code [snippet]="snippets.observableSetup.component"></ngbd-code>
+
+
+
+<h5>The basic <code>Observable</code> chain</h5>
+
+<p>
+  In this basic example we will simply use a statically defined and synchronously available set of data — color names:
+</p>
+
+<ngbd-code [snippet]="snippets.data"></ngbd-code>
+
+<p>
+  The function used to provide the <code>Observable</code> to the typeahead receives a single argument: another <code>Observable</code>. It spawns:
```

Comment:
This paragraph is still hard to read/understand, please check it if you have time

Reply:
Can you state what you consider being hard to read/understand? This is a highly personal opinion so I can't guess what is a true good form.